danger-kotlin-library/version.gradle

```diff
@@ -1,2 +1,2 @@
 group 'systems.danger'
-version '0.7.1'
+version '1.0-beta'
```

Comment:
It should be `1.0.0-beta` they should be three according to https://semver.org/

```
<valid semver> ::= <version core>
                 | <version core> "-" <pre-release>
                 | <version core> "+" <build>
                 | <version core> "-" <pre-release> "+" <build>

<version core> ::= <major> "." <minor> "." <patch>
```

Reply:
done in 0408ef2